XRTK.Lumin/Packages/com.xrtk.lumin/LuminPlatform.cs

```diff
@@ -9,14 +9,27 @@ namespace XRTK.Definitions.Platforms
     public class LuminPlatform : BasePlatform
     {
         /// <inheritdoc />
-        public override bool IsAvailable
+        public override bool IsActive
         {
             get
             {
 #if PLATFORM_LUMIN
                 return !UnityEngine.Application.isEditor;
 #else
                 return false;
+#endif
+            }
+        }
+
+        /// <inheritdoc />
+        public override bool IsBuildTargetAvailable
```

Comment:
Should probably be `IsBuildTargetActive` since Available sounds more like it's telling whether the Lumin module is installed for the editor instance.

Reply:
That name conflicts with am existing method in `CurrentTargetBuildPlatform`